experimental/Schemes/CoveredScheme.jl

```diff
@@ -101,6 +101,24 @@ affine_refinements(C::Covering) = C.affine_refinements
 ########################################################################
 
 @attr function standard_covering(X::AbsProjectiveScheme{<:Ring, <:MPolyQuoRing})
+  return _standard_covering(X, identity)
+end
+
+
+@attr function standard_covering(X::AbsProjectiveAlgebraicSet)
```

Comment:
Is there *any* type hint we could give for the return type? E.g.
```suggestion
@attr Covering function standard_covering(X::AbsProjectiveAlgebraicSet)
```
Same for the other uses of `@attr`

Reply:
@simonbrandhorst and I talked about this beforehand and I will overhaul this part in general.